staging/src/k8s.io/cri-api/pkg/OWNERS

```diff
@@ -0,0 +1,17 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# Disable inheritance as this is owned by sig-node (should mirror same contents as pkg/kubelet/OWNERS)
+options:
+  no_parent_owners: true
+approvers:
```

Comment:
I think the SIG node leads need to be in the OWNERS file, but not sure about the rest (since some of them are not as involved in CRI. 

@dchen1107 @derekwaynecarr thoughts?

Reply:
@yujuhong we are missing a `sig-node-approvers` in the OWNERS_ALIASES file. once we add that we can update the various sig-node related directories. We should not block this PR on that effort

Reply:
We can start with mirroring the same contents as pkg/kubelet/OWNERS, I think.

Reply:
Right this file (staging/src/k8s.io/cri-api/pkg/OWNERS) has the same contents of pkg/kubelet/OWNERS + the no_parent_owners: true option